lib/utils/encoding/sdkSuffix/getSDKSuffix.js

```diff
@@ -0,0 +1,52 @@
+let fs = require('fs');
+let path = require('path');
+let encodeVersion = require('./encodeVersion');
+
+let nodeSDKValue = 'M'; // Constant per SDK
+
+
+/**
+ * @description Turns x.y.z into x.y
+ * @param {string} semVerStr
+ */
+function removePatchFromSemver(semVerStr) {
+  let parts = semVerStr.split('.');
+
+  return `${parts[0]}.${parts[1]}`;
+}
+
+/**
+ * @description Gets the SDK signature by encoding the SDK version and node version
+ * @param {{isResponsive:boolean}} features
```

Comment:
Should be 'responsive' and not 'isResposnive'. 
How can anyone get the list of available features?

Reply:
There's currently only a single param in Node (responsive).
There's no way to get the complete set.

I decided to use isResponsive because it's a feature check, and it's a boolean.
This is passed as an argument into this function. 

I really want to avoid passing opaque 'configs' or 'options' around functions, or to use global variables (such as config).